### PR TITLE
Print date and time in log for requests.

### PR DESCRIPTION
**Pull request by andreynering**
I think such thing is indispensable. :smile:

### CODE REVIEW

File: logger.go [hunk 1]
```diff
@@ -23,6 +23,8 @@ import (
 	"time"
 )
 
+const LOG_TIME_FORMAT = "2006-01-02 15:04:05"
```
Comment (Unknwon) [inline logger.go]: I think this could be a variable, so user could change it with desired format.

**Comment by andreynering** [comment]
Done.

I couldn't find where `[Macaron]` is inserted in the log. Maybe there would be a better place for this, since it would be printed everywhere?

**Comment by Unknwon** [comment]
https://github.com/go-macaron/macaron/blob/master/macaron.go#L85

**Comment by Unknwon** [comment]
Thanks!

**Pull request merged**